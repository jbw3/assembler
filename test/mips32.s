label:
BEQ  t0, t1, label
BGEZ t2, label
BGTZ t2, label
BLEZ t2, label
BLTZ t2, label
BNE  t0, t1, label
