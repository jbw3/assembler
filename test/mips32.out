ffff0911
feff4105
fdff401d
fcff4019
fbff4005
faff0915
